templates/report_template.html

```diff
@@ -108,11 +111,19 @@
 							<td style="text-align: center;">{{row.build_days_ago}}</td>
 							<td style="text-align: center;">{{ composes(row) }}</td>
 							<td style="text-align: center;" bgcolor="#ef2929">{{row.lcb_result}}</td>
+							<td style="text-align: center;"><p>{{ row.stage_name }}</p>
+							{% if row.stage_urls is not none %}
+							{% for url in row.stage_urls %}
+							<br/>
+							<a href="{{ url }}">{{ url.split("/")[-1] }}</a>
+							{% endfor %}
+							{% endif %}
+							</td>
```

Comment:
What will show here if row.stage_urls is none?

Reply:
> What will show here if row.stage_urls is none?
Nothing, no urls will be added. This column will contain only stage_name.

Reply:
So, a blank square?

Reply:
Discussed on IRC